### PR TITLE
Nvss209 plus codesystems

### PR DESCRIPTION
**Pull request by saulakravitz**
STU2: Added Observation.dataAbsentReason to Resource Profile: Decedent’s age

### CODE REVIEW

File: VRDR/CodeSystems.cs [hunk 1]
```diff
@@ -19,7 +19,7 @@ public static class CodeSystems
         public static string PH_NullFlavor_HL7_V3 = "urn:oid:2.16.840.1.113883.5.1008";
         
         /// <summary>HL7 Data Absent reason.</summary>
-        public static string HL7_Data_Absent_Reason = "http://terminology.hl7.org/CodeSystem/data-absent-reason";     
+        public static string NullFlavor_HL7_V3 = "http://terminology.hl7.org/CodeSystem/data-absent-reason";
```
Comment (smacadam) [inline VRDR/CodeSystems.cs:22]: Is this name right?

**Review by smacadam — COMMENTED** [review]
Add a couple of tests to verify reading in a json and ije record and getting the expected values in the Age field.

File: VRDR/CodeSystems.cs (hunk 1 above)
Comment (saulakravitz) [reply VRDR/CodeSystems.cs:22]: Good catch....

File: VRDR/DeathRecord.cs [hunk 2]
```diff
@@ -5889,45 +5905,39 @@ public string DeathLocationDescription
...
-                    {
-                        quant.Value = Convert.ToDecimal(GetValue(value, "value"));
+
+                // If the value or unit is null, put out a data absent reason
+
+
+                if ( (!String.IsNullOrWhiteSpace(GetValue(value, "value"))) && !GetValue(value, "value").Equals("999") ){ // not unknown - NVSS-209
+                    String v = GetValue(value, "value");
+                    if (v.Equals("999")){
+                        AgeAtDeathObs.DataAbsentReason =  new CodeableConcept(CodeSystems.NullFlavor_HL7_V3, "unknown", "Unknown", null);
```
[the remark this answers is not in the document]
Comment (saulakravitz) [reply VRDR/DeathRecord.cs:5920]: Good catch.

File: VRDR/CodeSystems.cs (hunk 1 above)
Comment (saulakravitz) [reply VRDR/CodeSystems.cs:22]: Nope.

**Review by smacadam — APPROVED** [review]

**Pull request merged**